packages/core-components/src/lib/unsorted/ui/Databases/DuckdbForm.svelte

```diff
@@ -113,8 +113,9 @@
 				bind:value={ext}
 				on:change={handleKey}
 			>
-				<option value=".db">.db</option>
 				<option value=".duckdb">.duckdb</option>
+				<option value=".db">.db</option>
+				<option value="">No extension</option>
 			</select>
```

Comment:
This UI is not great.

Firstly it doesn't tell you what to do (it's telling you it needs a file, but in the case of motherduck it's a string)

Also Evidence really only works properly if you pass a service token (otherwise it tries to reauth for every query, which is unusable), and this would be much clearer if there was a separate box for it

Reply:
I wonder if the `MOTHERDUCK_TOKEN` works for the JS duckdb client library

Reply:
It does :)

<img width="970" alt="image" src="https://github.com/evidence-dev/evidence/assets/58074498/a24a7aba-9d08-4b3b-8eb5-3caec4b2c3de">

Reply:
Oh sorry I meant the env var that's referenced in the docs: https://motherduck.com/docs/authenticating-to-motherduck#saving-the-service-token-as-an-environment-variable

If that works then the user could simply type `md:` in that field and set the env var elsewhere.

Reply:
Hmm good q, not sure